src/main/java/org/folio/circulation/domain/UpdateRequestService.java

```diff
@@ -43,4 +52,28 @@ public UpdateRequestService(
 
     return succeeded(requestAndRelatedRecords);
   }
+
+  public Result<RequestAndRelatedRecords> sendNoticeOnRequestUpdated(
+    RequestAndRelatedRecords relatedRecords) {
+    if (relatedRecords.getRequest().getStatus() == RequestStatus.CLOSED_CANCELLED) {
+      sendNoticeOnRequestCancelled(relatedRecords);
+    }
+    return Result.succeeded(relatedRecords);
+  }
+
+  private void sendNoticeOnRequestCancelled(RequestAndRelatedRecords relatedRecords) {
+    Request request = relatedRecords.getRequest();
+    Item item = request.getItem();
+    User requester = request.getRequester();
+
+    PatronNoticeEvent requestCanceledEvent = new PatronNoticeEventBuilder()
```

Comment:
The trigger for this event is called `Request cancellation`, this variable is named `requestCanceledEvent`. The former spells cancellation with two `l`'s whereas the latter with a single `l`.

Why is the spelling different?

Reply:
Sorry, I missed that.  Fixed